src/components/searchSection/catergrayList.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import { MDBIcon } from 'mdbreact';
+import './myStyle.scss';
+
+function CatergrayList({ items }) {
```

Comment:
change this into arrow function
the code should be has a high level of consistency

Reply:
this the function name of the component how can i make it arrow ,we always make it like this

Reply:
Oh My bad
if it's a component function and you all used it like this so that's OK